reference/Events-API.yaml

```diff
@@ -0,0 +1,383 @@
+openapi: 3.0.1
+info:
+  title: Events API
+  version: 'ersion: DRAFT'
```

Comment:
skipped `v` on purpose. To get Stoplight dispay tag "version: DRAFT"

<img width="261" alt="Screenshot 2024-03-07 at 09 57 11" src="https://github.com/StreamAMG/cloudmatrix-documentation-portal/assets/9783762/101fbb15-7a38-4aed-9de6-8e971ea1459b">

Reply:
found a better way thourgh Jon's previous work and unified the files :)